templates/tease-search.twig

```diff
@@ -64,7 +64,7 @@
 				</div>
 			</div>
 
-			<p class="search-result-item-content">{{ post.preview().read_more('')|excerpt( 25 )|raw }}</p>
+			<p class="search-result-item-content">{{ post.post_excerpt|excerpt( 25 )|e('wp_kses_post')|raw }}</p>
```

Comment:
We need to check if this re-introduces that weird bug we used to have in the past when the actual excerpt was less than 25 words. The `.read_more('')` was added to solve that issue which caused the next search results to appear misaligned with broken markup.

Reply:
Oh, didn't know about that, will check it further then.